rs/nervous_system/clients/src/canister_status.rs

```diff
@@ -66,6 +66,7 @@ pub struct DefiniteCanisterSettings {
     pub reserved_cycles_limit: Option<candid::Nat>,
     pub wasm_memory_limit: Option<candid::Nat>,
     pub log_visibility: Option<LogVisibility>,
+    pub wasm_memory_threshold: Option<candid::Nat>,
```

Comment:
We now have a few limits / thresholds in these settings; would it make sense to add a doc string to explain the difference / link to the interface spec?

Reply:
That makes sense, I added a comment.